staging/src/github.com/kubeedge/api/apidoc/generated/openapi/zz_generated.openapi.go

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
why do we have a difference in here? which is not supposed to be modified manually?